examples/embeddable_examples/public/react_embeddables/saved_book/create_saved_book_action.tsx

```diff
@@ -42,7 +41,7 @@ export const registerCreateSavedBookAction = (uiActions: UiActionsPublicStart, c
         core,
       });
 
-      const bookAttributes = serializeBookAttributes(newPanelStateManager);
+      const bookAttributes = newPanelStateManager.getLatestState();
       const initialState: BookRuntimeState = savedBookId
         ? { savedBookId, ...bookAttributes }
         : { ...bookAttributes };
```

Comment:
I could be wrong, but you might need to update the arguments in `embeddable.addNewPanel` below this line as I believe the interface has changed from `initialState` to `serializedState: { rawState: initialState }`.